app/assets/stylesheets/_sidebar.scss

```diff
@@ -55,9 +55,10 @@
   &-details {
     @include padding($base-spacing);
     @include span-columns(9 of 12);
-    border: $base-border;
     bottom: 0;
     position: absolute;
     top: $actions-height;
+    background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.